extensions/dicom-video/src/viewports/OHIFCornerstoneVideoViewport.tsx

```diff
@@ -1,26 +1,47 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 
-function OHIFCornerstoneVideoViewport({ displaySets }) {
+function OHIFCornerstoneVideoViewport({ displaySets, servicesManager }) {
```

Comment:
const { servicesManager } = useServices() is the preferred way to get at the services manager.